src/Uno.UI/Extensions/TextAlignmentExtensions.cs

```diff
@@ -37,9 +37,12 @@ internal static AppKit.NSTextAlignment ToNativeTextAlignment(this TextAlignment
 					return AppKit.NSTextAlignment.Right;
 				case TextAlignment.Justify:
 					return AppKit.NSTextAlignment.Justified;
-				default:
 				case TextAlignment.Left:
 					return AppKit.NSTextAlignment.Left;
+				case TextAlignment.DetectFromContent:
+					return AppKit.NSTextAlignment.Natural;
```

Comment:
Is this the correct native value for `DetectFromContent`? Previously it was `Left`, which I assume is incorrect since `TextAlignment.Left` shouldn't be equivalent to `DetectFromContent`.

Reply:
@ajpinedam would you know about this one ?

Reply:
@Youssef1313 You can always check the screenshots in the CI ;-)  But that's not an optimal development process.

Reply:
Yes this `AppKit.NSTextAlignment.Natural` is the right one that matches the `TextAlignment.DetectFromContent`. In both cases, the text will be aligned depending on the Culture specified in the program.